src/main/java/sirius/biz/analytics/scheduler/BaseEntityBatchEmitter.java

```diff
@@ -118,7 +118,7 @@
 
         Class<E> type = (Class<E>) mixing.getDescriptor(typeName).getType();
         Q query = (Q) getMapper().select(type);
-        query.where(getMapper().filters().gte(BaseEntity.ID, startId))
+        query.where(getMapper().filters().gt(BaseEntity.ID, startId))
```

Comment:
shouldn't it be inverted: >= startId and < endId ?

Reply:
nope - should be good (??)

Reply:
In the first run, how do we reach the first Id?

Reply:
in the first run startId is null or "" and thus the constraint is ignored...